Resources/config/services.xml

```diff
@@ -29,37 +28,29 @@
+            <tag name="templating.helper" alias="phone_number_helper"/>
+            <!-- phone_number_format is deprecated and will be removed in 2.0 (use phone_number_helper instead) -->
             <tag name="templating.helper" alias="phone_number_format"/>
             <argument type="service" id="libphonenumber.phone_number_util"/>
-
         </service>
 
-        <service id="misd_phone_number.twig.extension.format" class="Misd\PhoneNumberBundle\Twig\Extension\PhoneNumberFormatExtension"
-                 public="false">
+        <service id="misd_phone_number.twig.extension.format" class="Misd\PhoneNumberBundle\Twig\Extension\PhoneNumberHelperExtension" public="false">
             <tag name="twig.extension"/>
-            <argument type="service" id="misd_phone_number.templating.helper.format"/>
+            <argument type="service" id="misd_phone_number.templating.helper"/>
         </service>
 
         <service id="misd_phone_number.form.type" class="Misd\PhoneNumberBundle\Form\Type\PhoneNumberType">
             <tag name="form.type" alias="tel"/>
         </service>
 
         <service id="misd_phone_number.serializer.handler" class="Misd\PhoneNumberBundle\Serializer\Handler\PhoneNumberHandler">
-            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization"
-                 format="json" method="serializePhoneNumber"/>
-            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="deserialization"
-                 format="json" method="deserializePhoneNumberFromJson"/>
-            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization"
-                 format="xml" method="serializePhoneNumber"/>
-            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="deserialization"
-                 format="xml" method="deserializePhoneNumberFromXml"/>
-            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization"
-                 format="yml" method="serializePhoneNumber"/>
+            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization" format="json" method="serializePhoneNumber"/>
+            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="deserialization" format="json" method="deserializePhoneNumberFromJson"/>
+            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization" format="xml" method="serializePhoneNumber"/>
+            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="deserialization" format="xml" method="deserializePhoneNumberFromXml"/>
+            <tag name="jms_serializer.handler" type="libphonenumber\PhoneNumber" direction="serialization" format="yml" method="serializePhoneNumber"/>
```

Comment:
Revert these, nothing has changed and it makes the diff hard to read. (There are lots of other non-changes that need reverting too, eg new line removal.)